src/webview/apps/lightspeed/playbookGeneration/main.ts

```diff
@@ -57,11 +58,16 @@ window.addEventListener("message", (event) => {
       changeDisplay("firstMessage", "none");
       changeDisplay("secondMessage", "block");
       changeDisplay("generatePlaybookContainer", "block");
+      changeDisplay("promptContainer", "block");
 
       const element = document.getElementById("playbook-text-area") as TextArea;
-      savedSummary = element.value = message.summary;
+      savedSummary = element.value = message.summary.content;
```

Comment:
There is no `.content` key. `message.summary` is a string.

Reply:
You're right. It was a part of #1301... I will get rid of that. Thanks.